src/Test/AbstractWidgetTestCase.php

```diff
@@ -33,42 +39,67 @@
 abstract class AbstractWidgetTestCase extends TypeTestCase
 {
     /**
-     * @var FormRenderer
+     * @var FormExtensionInterface
+     */
+    private $extension;
+
+    /**
+     * @var TwigRenderer
      */
     private $renderer;
 
     protected function setUp(): void
     {
         parent::setUp();
 
-        $environment = $this->getEnvironment();
-
-        $this->renderer = new FormRenderer(
-            $this->getRenderingEngine($environment),
-            $this->createMock(CsrfTokenManagerInterface::class)
-        );
-
-        $environment->addRuntimeLoader(new FactoryRuntimeLoader([
-            FormRenderer::class => function () {
-                return $this->renderer;
-            },
-        ]));
-        $environment->addExtension(new FormExtension());
+        // TODO: remove the condition when dropping symfony/twig-bundle < 3.2
```

Comment:
Since you are dropping twig-bundle 3.2 (min version is 4.3 now, we can drop some of this code, right?